<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ParamSettingForm</class>
 <widget class="QWidget" name="ParamSettingForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(72,61, 139);</string>
  </property>
  <widget class="QPushButton" name="printSettingBut">
   <property name="geometry">
    <rect>
     <x>4</x>
     <y>70</y>
     <width>90</width>
     <height>90</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>楷体</family>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/Images/printSetting.bmp);
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>打印设置</string>
   </property>
  </widget>
  <widget class="QPushButton" name="sysSettingBut">
   <property name="geometry">
    <rect>
     <x>4</x>
     <y>190</y>
     <width>90</width>
     <height>90</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>楷体</family>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/Images/sysSetting.bmp);
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>系统设置</string>
   </property>
  </widget>
  <widget class="QPushButton" name="countSettingBut">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>330</y>
     <width>0</width>
     <height>0</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>楷体</family>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/Images/countSetting.bmp);
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>计数设置</string>
   </property>
  </widget>
  <widget class="QPushButton" name="aboutMacBut">
   <property name="geometry">
    <rect>
     <x>4</x>
     <y>310</y>
     <width>90</width>
     <height>90</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>楷体</family>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/Images/aboutMac.bmp);
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>关于本机</string>
   </property>
  </widget>
  <widget class="QStackedWidget" name="paraManageStacWid">
   <property name="geometry">
    <rect>
     <x>98</x>
     <y>0</y>
     <width>926</width>
     <height>591</height>
    </rect>
   </property>
   <widget class="QWidget" name="page"/>
   <widget class="QWidget" name="page_2"/>
  </widget>
  <widget class="QPushButton" name="paraExitBut">
   <property name="geometry">
    <rect>
     <x>4</x>
     <y>430</y>
     <width>90</width>
     <height>90</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>楷体</family>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/Images/exit.bmp);
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>退出</string>
   </property>
  </widget>
  <widget class="QLabel" name="paraManageBackLabel">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1024</width>
     <height>600</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(72,61, 139);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="resetBut">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>540</y>
     <width>111</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>楷体</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(0, 0, 230);</string>
   </property>
   <property name="text">
    <string>重置</string>
   </property>
  </widget>
  <widget class="QPushButton" name="savePrintSetBut">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>540</y>
     <width>111</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>楷体</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(0, 0, 230);</string>
   </property>
   <property name="text">
    <string>保存</string>
   </property>
  </widget>
  <zorder>paraManageBackLabel</zorder>
  <zorder>printSettingBut</zorder>
  <zorder>sysSettingBut</zorder>
  <zorder>countSettingBut</zorder>
  <zorder>aboutMacBut</zorder>
  <zorder>paraExitBut</zorder>
  <zorder>paraManageStacWid</zorder>
  <zorder>resetBut</zorder>
  <zorder>savePrintSetBut</zorder>
 </widget>
 <resources>
  <include location="Res.qrc"/>
 </resources>
 <connections/>
</ui>
